src/AssemblyAI/Lemur/Requests/LemurQuestionAnswerParams.cs

```diff
@@ -5,11 +5,11 @@
 
 namespace AssemblyAI;
```

Comment:
Namespace should be `AssemblyAI.Lemur`, and consistently named for other classes.

Reply:
I've made myself a ticket for this to track, but will need more time to look into this

hopefully this is ok for now since there's no change being made here?

Reply:
Yes, but this will be a breaking change, so we won't release this as GA until implemented.

Reply:
ack, will look into with high priority

Reply:
I'm proposing the following changes internally to our spec:

- `Error` schema is now `ApiError`
- `error` property is now `error_message`
- `TerminateSession` message and schema is now `TerminateSessionMessage`

Reply:
Hey @Swimburger can you explain the motivation for the namespace change? With the current namespace convention, we minimize the imports required since most types are found in the base namespace

Reply:
I'm worried about name collisions. Especially in the future where we may have a v3 transcript that will be very different from v2.

Reply:
Makes sense, i'm going to work on adding a config flag for C# that enable this namespace scheme for you 👍